lib/InitAll.cpp

```diff
@@ -20,6 +21,7 @@
 #include "torch-mlir/RefBackend/Passes.h"
 
 void mlir::torch::registerAllDialects(mlir::DialectRegistry &registry) {
+  registry.insert<mlir::func::FuncDialect>();
```

Comment:
@silvasean are there any problems with registering this dialect here?

Reply:
I think it is okay